SupportSDKSamples/CocoaPodSample/ViewController.swift

```diff
@@ -0,0 +1,169 @@
+//
+//  ViewController.swift
+//  CocoaPodSample
+//
+//  Created by Killian Smith on 22/05/2018.
+//  Copyright © 2018 Killian Smith. All rights reserved.
+//
+
+import UIKit
+
+import ZendeskSDK
+import ZendeskCoreSDK
+
+class ViewController: UIViewController, UINavigationControllerDelegate {
+    
+    let imagePickerController = UIImagePickerController()
+    private let toastWrapper = ZDKToastViewWrapper()
+    
+    // MARK: Outlets
+    @IBOutlet weak var subjectTextField: UITextField!
+    @IBOutlet weak var descriptionTextView: UITextView!
+    @IBOutlet weak var attachmentImageView: UIImageView!
+    @IBOutlet weak var ticketListButton: UIButton!
+    @IBOutlet weak var sendButton: UIButton!
+    @IBOutlet weak var addImageButton: UIButton!
+    @IBOutlet weak var customFieldTextField: UITextField!
+    
+    var hcConfig: HelpCenterUiConfiguration {
+        let hcConfig = HelpCenterUiConfiguration()
+        hcConfig.hideContactSupport = true
+        hcConfig.labels = ["Breakout"]
+        return hcConfig
+    }
+    
+    var requestConfig: RequestUiConfiguration {
+        let config = RequestUiConfiguration()
+        config.tags = ["Some","stupid", "tags"]
```

Comment:
tags are actually one of the first points of customisation provided to our customers. Try to avoid using `stupid` wording. 
Rather name the tags in generic way, e.g. `tag1`, `tag2`, etc